docs/dal/index.rst

```diff
@@ -319,35 +358,49 @@ datasets and/or access to additional data services.
 
 To obtain the names of the columns in a service response, write:
 
+.. doctest-skip::
+.. doctest-remote-data::
+
```

Comment:
Please advise how `resultset` should be defined to make sense for the examples below. Tests are temporarily skipped until that definition is in place to make them pass.

Reply:
I don't know which table would contain `accref`, but we could use either the first TAP example or the SSA example with one of the fields in there:
```
tap_service = vo.dal.TAPService("http://dc.g-vo.org/tap")
resultset = tap_service.search("SELECT TOP 10 * FROM ivoa.obscore")
print(resultset.fieldnames)
```
and pick any interesting field from there to replace `accref` with and everything should work.